Lps 205412

lfbesada opened the pull request:
:kissing_heart:

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

ealonso committed:
LPS-205412 Calculate pageContents only if it is necessary

sandrodw3 committed:
LPS-205412 Add PageContent type to types folder

sandrodw3 committed:
LPS-205412 Instead of updating page contents, set them to null

This way, they will be calculated again the next time they are needed

sandrodw3 committed:
LPS-205412 Types

sandrodw3 committed:
LPS-205412 Add hook to ask the backend for the page contents if they are not in the state

sandrodw3 committed:
LPS-205412 Change initial state to null

In the case of UPDATE_PREVIEW_IMAGE, that action can only be triggered from the Page Contents panel itself, so we know they will exist

sandrodw3 committed:
LPS-205412 Use hook instead of selector

sandrodw3 committed:
LPS-205412 Adapt tests

lfbesada committed:
LPS-205412 Adapt Integration tests
